pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -56,7 +56,6 @@ import (
 
 	cmdv1 "kubevirt.io/kubevirt/pkg/handler-launcher-com/cmd/v1"
 	kubevirtpointer "kubevirt.io/kubevirt/pkg/pointer"
-	virtpointer "kubevirt.io/kubevirt/pkg/pointer"
```

Comment:
Can you please split this formatting/styling changes in a separate commit?

Reply:
I have split the formatting changes into a separate commit.